catroid/src/main/java/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.java

```diff
@@ -822,7 +829,9 @@ public String getBrickCategory(Brick brick, boolean isBackgroundSprite, Context
 		} else if (brick instanceof SetVariableBrick) {
 			category = res.getString(R.string.category_data);
 		} else if (brick instanceof WebRequestBrick) {
-			category = res.getString(R.string.category_control);
+			category = res.getString(R.string.category_data);
```

Comment:
why is here a reference to the webrequestbrick?

Reply:
in the original commit it was missed that the webrequest brick also needs his referenze updated (help-address), I could remove that part and file a bug if that is prefered

Reply:
no thats okey :)